### DISCUSSION TITLE
Need a good first issue to get started!

### DISCUSSION THREAD

**Discussion by ZEUS-03** [EddieHubCommunity/LinkFree#6230] [General]
Hey! I see some good first issues but they are all assigned. Need an entry point to start contributing to the LinkFree community and also successfully submit my first pull request. Will be pleased to contribute to the LinkFree community.

**Comment by geeky-sambhav** [comment]
> Hey! I see some good first issues but they are all assigned. Need an entry point to start contributing to the LinkFree community and also successfully submit my first pull request. Will be pleased to contribute to the LinkFree community.

I really understand your concern because at some point I was also facing the same  problem as u are facing  But what I did was to browse the linkfree repo every now and then until I found any good first issue which was unassigned.The best way to do is to install github app in your mobile and just like you scroll your Instagram, scroll through the repo for unassigned good first issue.

Alternatively If you don't find any suitable issues, you could also consider contributing to the documentation. Improving documentation is a great way to contribute to open source projects and is often a good starting point for new contributors.

**Reply by mintr-dev** [reply]
@geeky-sambhav Truly said , Everyone's first experience in open-source is somewhat like this

**Reply by ZEUS-03** [reply]
Got it. Thanks!
